04-store-api/starter/app.js

```diff
@@ -1 +1,47 @@
 console.log('04 Store API')
+
+
+require('dotenv').config();
+require('express-async-errors');
+
+const express = require('express');
+const app = express();
+
+const connectDB = require('./db/connect');
+const productsRouter = require('./routes/products');
+
+const notFoundMiddleware = require('./middleware/not-found');
+const errorMiddleware = require('./middleware/error-handler');
+
+// middleware
+app.use(express.json());
+
+// routes
+
+app.get('/', (req, res) => {
+  res.send('<h1>Store API</h1><a href="/api/v1/products">products route</a>');
+});
+
+app.use('/api/v1/products', productsRouter);
+
+// products route
+
+app.use(notFoundMiddleware);
+app.use(errorMiddleware);
+
+const port = process.env.PORT || 3000;
+
+const start = async () => {
+  try {
+    // connectDB
+    await connectDB(process.env.MONGO_URI)
+    .then(() => console.log('MongoDB connected'));
```

Comment:
You don't need to use `then` in Promises when you use `await`. You can just do the `console.log` right below the `connectDB` call.

If you weren't using `async await`, you would need the `then` to ensure it only happened after the connection was established.

```suggestion
    await connectDB(process.env.MONGO_URI)
    console.log('MongoDB connected');
```